### ISSUE TITLE
RestrictedCreative v1.15.3 not loading again

### ISSUE THREAD

**Issue by andris155** [Prunt/RestrictedCreative#43]
```
[10:29:03] [Server thread/INFO]: [RestrictedCreative] Enabling RestrictedCreative v1.15.3
[10:29:04] [Server thread/ERROR]: Error occurred while enabling RestrictedCreative v1.15.3 (Is it up to date?)
java.lang.NullPointerException: null
	at me.prunt.restrictedcreative.Main.getMessage(Main.java:726) ~[?:?]
	at me.prunt.restrictedcreative.Main.loadConfig(Main.java:645) ~[?:?]
	at me.prunt.restrictedcreative.Main.onEnable(Main.java:171) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[patched_1.12.2.jar:git-Paper-1523]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316) ~[patched_1.12.2.jar:git-Paper-1523]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) ~[patched_1.12.2.jar:git-Paper-1523]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:395) ~[patched_1.12.2.jar:git-Paper-1523]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:344) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.MinecraftServer.t(MinecraftServer.java:442) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.MinecraftServer.l(MinecraftServer.java:403) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.MinecraftServer.a(MinecraftServer.java:341) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:289) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:616) ~[patched_1.12.2.jar:git-Paper-1523]
	at java.lang.Thread.run(Unknown Source) [?:?]
[10:29:04] [Server thread/INFO]: [RestrictedCreative] Disabling RestrictedCreative v1.15.3
[10:29:04] [Server thread/ERROR]: Error occurred while disabling RestrictedCreative v1.15.3 (Is it up to date?)
java.lang.NullPointerException: null
	at me.prunt.restrictedcreative.Main.onDisable(Main.java:600) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:266) ~[patched_1.12.2.jar:git-Paper-1523]
	at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:350) ~[patched_1.12.2.jar:git-Paper-1523]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:320) ~[patched_1.12.2.jar:git-Paper-1523]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) ~[patched_1.12.2.jar:git-Paper-1523]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:395) ~[patched_1.12.2.jar:git-Paper-1523]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:344) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.MinecraftServer.t(MinecraftServer.java:442) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.MinecraftServer.l(MinecraftServer.java:403) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.MinecraftServer.a(MinecraftServer.java:341) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:289) ~[patched_1.12.2.jar:git-Paper-1523]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:616) ~[patched_1.12.2.jar:git-Paper-1523]
	at java.lang.Thread.run(Unknown Source) [?:?]
```

**Comment by Prunt** [comment]
Are you sure you have the latest-latest version? This is the exact error I fixed in the update I uploaded yesterday, but didn't change the version number.

**Comment by andris155** [comment]
Yes. This is the latest version.

**Comment by Prunt** [comment]
I just edited my comment as you posted your comment, so in case you missed it:
> Other than that, simply adding "block-break" under "messages" in the config should do the trick. Check the default config for reference.

**Comment by andris155** [comment]
I tested default with default config file. But not work.

**Comment by Prunt** [comment]
Turns out that if the version string isn't changed, Spigot won't post a new file with the update. That means you didn't get the updated file when you downloaded the latest version. I'll be adding some other fixes to the new version (1.15.4) as well, so you should be able to use that soon.
